contracts/RiskManagerV1.sol

```diff
@@ -45,6 +47,16 @@ contract RiskManagerV1 is Auctioneer, Ownable {
 
     uint256 public constant DEPOSIT_LIQUIDATION_IN_PROGRESS_STATE = 10;
     uint256 public constant DEPOSIT_LIQUIDATED_STATE = 11;
+    // Auction will not be opened if the deposit collateralization level does not
+    // fall below this collateralization threshold.
+    // Risk manager should open a coverage pool auction for only those deposits
+    // that nobody else is willing to purchase. The default value can be updated
+    // by the governance in two steps. First step is to begin the update process
+    // with the new value and the second step is to finalize it after
+    // GOVERNANCE_TIME_DELAY has passed.
+    uint256 public collateralizationThreshold = 101; // percent
```

Comment:
I'd consider making it a constructor parameter, just like `_auctionLength` for consistency. It should also give us a chance to adjust this value last minute before the deployment without having to change the code.